poc_mobile_verifier/src/cli/reward_from_db.rs

```diff
@@ -0,0 +1,70 @@
+use crate::{
+    env_var, heartbeats::Heartbeats, speedtests::SpeedtestAverages,
+    subnetwork_rewards::SubnetworkRewards, Result,
+};
+use chrono::{DateTime, NaiveDateTime, Utc};
+use helium_crypto::PublicKey;
+use helium_proto::services::{follower, Endpoint, Uri};
+use serde_json::json;
+use sqlx::postgres::PgPoolOptions;
+
+use super::{CONNECT_TIMEOUT, DEFAULT_URI, RPC_TIMEOUT};
+
+/// Reward a period from the entries in the database
+#[derive(Debug, clap::Args)]
+pub struct Cmd {
+    #[clap(long)]
+    start: NaiveDateTime,
+    #[clap(long)]
+    end: NaiveDateTime,
+}
+
+impl Cmd {
+    pub async fn run(self) -> Result {
+        let Self { start, end } = self;
+
+        let start = DateTime::from_utc(start, Utc);
```

Comment:
any reason to pattern match the start and end out above vs. `self.start` and `self.end` here or just preference?

Reply:
just preference